### PR TITLE
BookStore

### PR DESCRIPTION
**Pull request by AhmadiGu**
### Components initialization

- add setup
- add books component
- add the form to add books
- add the navbar to toggle pages

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const AddBook = () => (
+  <div>
+    <h2 className="form-title"> ADD NEW BOOK</h2>
+    <form className="add-form">
+      <input className="form-item add-title" type="text" placeholder="Book title" />
```
Comment (muneebulrehman) [inline src/components/AddBook.js:7]: The form in the `AddBook` component should have input for the name of the author as well. The category part is optional. But the name of the author is a part of the requirements for this project. 
image

File: README.md [hunk 2]
```diff
@@ -1,70 +1,157 @@
...
+ 
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (muneebulrehman) [inline README.md]: All the main headers from this [README template](https://github.com/microverseinc/readme-template) should be present in a student's repo. The only optional section is a FAQ. Make sure the student has added the section **Future Features**, including 1-3 potential features to be added.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @AhmadiGu , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @AhmadiGu 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ST3D


## Highlights
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**